lib/mustache-parser.pegjs

```diff
@@ -4,15 +4,16 @@ start = newMustache
 // {
 //   name: string,
 //   attrs: array, (items look like 'class="abc"', 'tagName="xyz"', '(query-params foo=bar)', 'blah', '"quotedBlah"', etc)
+//   blockParams: array, (items are all strings like 'item', 'foo', etc.)
 //   modifier: '?' or '!' (optional),
 // }
 // upstream parsers will optionally add an `isEscaped:true` property
-newMustache = name:newMustacheStart m_* attrs:(m_bracketedAttrs / newMustacheAttr*) {
+newMustache = name:newMustacheStart m_* attrs:(m_bracketedAttrs / newMustacheAttr*) blockParams:m_blockParams? {
```

Comment:
@mixonic actually you lost me slightly.  Are you referring to the return value  of the `newMustache` rule in your comment?